<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveLoadDialog</class>
 <widget class="QDialog" name="SaveLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Laden/Speichern</string>
  </property>
  <property name="windowIcon">
   <iconset theme="document-save"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ausleihe:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="BookSelectPathButton">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="CDSelectPathButton">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Bitte Pfad zu den Listen angeben!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Personen:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3" alignment="Qt::AlignmentFlag::AlignRight">
    <widget class="QDialogButtonBox" name="SaveLoadOKAbortButtons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="BookPathTextBox"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="CDPathTextBox"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="LendSelectPathButton">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLineEdit" name="PersonPathTextBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>BÃ¼cher:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CDs:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="PersonSelectPathButton">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="LendPathTextBox"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>LendPathTextBox</tabstop>
  <tabstop>BookPathTextBox</tabstop>
  <tabstop>CDPathTextBox</tabstop>
  <tabstop>PersonPathTextBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>SaveLoadOKAbortButtons</sender>
   <signal>accepted()</signal>
   <receiver>SaveLoadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveLoadOKAbortButtons</sender>
   <signal>rejected()</signal>
   <receiver>SaveLoadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
